pandas/core/ops.py

```diff
@@ -744,6 +744,13 @@ def wrapper(left, right, name=name, na_op=na_op):
             return NotImplemented
 
         left, right = _align_method_SERIES(left, right)
+        if is_datetime64_dtype(left) or is_datetime64tz_dtype(left):
```

Comment:
and until / unless you want to limit / remove _TimeOp (which is actually ok with me). then this doesn't belong here as I have commented before.

You are welcome to put it *in* _TimeOp for now or rip out TimeOp.